src/Flunt.Br/Validations/CarLicensePlate.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Flunt.Br.Document.Interfaces;
+using System.Text.RegularExpressions;
+
+namespace Flunt.Br.Validations
+{
+    internal class CarLicensePlate : IValidate
+    {
+        public bool Validate(string value)
+        {
+            return value.Length == 7 && new Regex(@"[A-Z]{3}[0-9][A-Z0-9][0-9]{2}", RegexOptions.Singleline).Match(value).Success;
```

Comment:
@luturol acredito que seja interessante também considerarmos válida uma placa que venha com hífen. Concorda?

Reply:
@lira92  Confesso que não tinha pensado nessa possibilidade. Irei fazer essas modificações mais tarde. Adicionar na Regex ou fazer um Replace do hífen para remover ele? Qual opção acha melhor?

Reply:
@luturol acho que as duas resolvem, quanto a remover o hífen, se for essa abordagem que escolher, cuide pra não alterar o estado do parâmetro, garanta que o hífen só é removido em alguma variável local do método ou algo do gênero.

Reply:
@lira92 Feito.